biz.aQute.bndlib/src/aQute/bnd/cdi/Discover.java

```diff
@@ -8,8 +8,14 @@
 	annotated_by_bean,
 	none;
 
+	static Discover parse(String s) {
+		if (s == null || s.isEmpty())
```

Comment:
You may want to do:

https://github.com/bndtools/bnd/blob/57a70d6a05d77289e2f4702246ece22984b89070/aQute.libg/src/aQute/lib/strings/Strings.java#L147

since it also trims the string in case the input value has leading or trailing whitespace.